src/css/base/typography.scss

```diff
@@ -18,6 +18,22 @@ a {
   &:active {
     color: $color-blue;
   }
+
+  /* stylelint-disable selector-no-qualifying-type */
+  // needed to target <a> styles
+  &.link-textblack {
```

Comment:
Does the rule need to be `a.link-textblack` or can it just be `.link-textblack` on its own? Or is the former being overridden by something else?